### PR TITLE
Cmd wget

### PR DESCRIPTION
**Pull request by AgnieszkaBylica**

### CODE REVIEW

**Comment by emcek** [comment]
I prefer to remove `print` statements or replace them with `logger` in `moler/io/raw/terminal.py` module

**Comment by Ernold11** [comment]
Please remove `print`. Do not adding logger in `terminal.py` there is good logging mechanism in higher layer of moler.

**Comment by AgnieszkaBylica** [comment]
Sorry, I forgot to delete prints after debugging.

**Comment by AgnieszkaBylica** [comment]
runner added

**Comment by Ernold11** [comment]
Was it test on real connection? You are sure that percentage changes go in new lines?

**Comment by AgnieszkaBylica** [comment]
Fixed after meeting: there was a mistake in regex but output lines are normal lines, there is no weird formatting.

**Comment by Ernold11** [comment]
Please merge current master into your branch.

**Comment by AgnieszkaBylica** [comment]
Done.

**Comment by AgnieszkaBylica** [comment]
Fixed usage of pytest fixture.

File: moler/cmd/unix/wget.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+class Wget(GenericUnixCommand):
+    def __init__(self, connection, options, log_progress_bar=False, timeout=60, prompt=None, newline_chars=None,
+                 runner=None):
+        super(Wget, self).__init__(connection=connection, prompt=prompt, newline_chars=newline_chars, runner=runner)
+
+        self.options = options  # should contain URLs
+        self.log_progress_bar = log_progress_bar
+        self.extend_timeout(timeout)
+        self.current_percent = 0
+        self.next_percent = 0
```
Comment (marcin-usielski) [inline moler/cmd/unix/wget.py]: Unused variable?

File: moler/cmd/unix/wget.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+
+from moler.cmd.unix.genericunix import GenericUnixCommand
+from moler.exceptions import CommandFailure
+from moler.exceptions import ParsingDone
+import re
+
+
+class Wget(GenericUnixCommand):
+    def __init__(self, connection, options, log_progress_bar=False, timeout=60, prompt=None, newline_chars=None,
```
Comment (marcin-usielski) [inline moler/cmd/unix/wget.py]: It's not place for timeout. Timeout is set when command starts running.

File: moler/cmd/unix/wget.py [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                 runner=None):
+        super(Wget, self).__init__(connection=connection, prompt=prompt, newline_chars=newline_chars, runner=runner)
+
+        self.options = options  # should contain URLs
+        self.log_progress_bar = log_progress_bar
+        self.extend_timeout(timeout)
```
Comment (marcin-usielski) [inline moler/cmd/unix/wget.py]: Wrong usage.

**Pull request merged**